src/main.rs

```diff
@@ -54,8 +54,9 @@ fn main() {
 
     match res {
         Ok(timing_results) => {
-            write_benchmark_comparison(&timing_results);
-            let ans = export_manager.write_results(timing_results, options.unwrap().time_unit);
+            let unwrapped = options.unwrap();
```

Comment:
instead of calling it `unwrapped`, we could call it `options` and shadow the original name.

The fact that we call `unwrap` here anyway is not really great, but that should be fixed elsewhere.

Reply:
updated with shodowing for now. do you have any ideas on re-writing in the way not to have to `unwrap` the `options` object?